ts/store/reducers/backendStatus/backendInfo.ts

```diff
@@ -0,0 +1,50 @@
+import { getType } from "typesafe-actions";
+import * as O from "fp-ts/lib/Option";
+import { pipe } from "fp-ts/lib/function";
+import { Action } from "../../actions/types";
+import { backendStatusLoadSuccess } from "../../actions/backendStatus";
+import { BackendStatus } from "../../../../definitions/content/BackendStatus";
+import { GlobalState } from "../types";
+
+// systems could be consider dead when we have no updates for at least DEAD_COUNTER_THRESHOLD times
+export const DEAD_COUNTER_THRESHOLD = 2;
+
+export type BackedInfoState = {
+  backendInfoMessage: O.Option<BackendStatus["message"]>;
+  areSystemsDead: boolean;
+  deadsCounter: number;
+};
+
+const initialBackendInfoState: BackedInfoState = {
+  backendInfoMessage: O.none,
+  areSystemsDead: false,
+  deadsCounter: 0
+};
+
+export const backendInfoReducer = (
+  state: BackedInfoState = initialBackendInfoState,
+  action: Action
+): BackedInfoState => {
+  if (action.type === getType(backendStatusLoadSuccess)) {
+    const deadsCounter =
+      action.payload.is_alive === false
+        ? Math.min(state.deadsCounter + 1, DEAD_COUNTER_THRESHOLD)
+        : 0;
+    return {
+      ...state,
+      backendInfoMessage: O.some(action.payload.message),
+      areSystemsDead: deadsCounter >= DEAD_COUNTER_THRESHOLD,
+      deadsCounter
+    };
+  }
+  return state;
+};
+
+export const backendInfoSelector = (state: GlobalState) => state.backendInfo;
```

Comment:
Since this selector is "a thing of the past" and it is used only in `ts/sagas/backendStatus.ts`, should we remove it and refactor the latter to use `isBackendServicesStatusOffSelector` and a new `deadsCounterSelector`?

Reply:
maybe would face this in a more dedicated task, I would prefer to avoid side effects due to un-needed refactoring at the moment, what do you think?

Reply:
I think it is a very simple and side-effect-free refactoring:

In the reducer, we remove the selector and add the new one:
`export const deadsCounterSelector = (state: GlobalState) => state.backendInfo.deadsCounter`

and in the saga, we replace the call to the old selector with two different calls
```
const areSystemsDead = yield* select(isBackendServicesStatusOffSelector)
...
if (areSystemsDead) { 
  ...
}

const deadsCounter = yield* select(deadsCounterSelector);
const sleepTime = deadsCounter > 0 && areSystemsDead === false ? ...
...
```

Since it is a saga, there is not even the risk of introducing re-renderings

Reply:
done in [9dd4cef](https://github.com/pagopa/io-app/pull/6296/commits/9dd4cef4ba6b36f30e75a63e0bfe5d55c52cb017)

Reply:
I think we may have introduced a redundant selector 
```
export const areSystemsDeadSelector = (state: GlobalState) =>
  state.backendInfo.deadsCounter;
```
since it does the same as `deadsCounterSelector` and what we want to use (together with the former) is
`isBackendServicesStatusOffSelector`

Reply:
just a copy and paste error, anyway was not needed.

removed in [6934ec2](https://github.com/pagopa/io-app/pull/6296/commits/6934ec2b2fae727dd8c812227c1bebc1d8b29e03)